lib/ansible/modules/windows/win_group_membership.py

```diff
@@ -35,8 +35,10 @@
   state:
     description:
       - Desired state of the members in the group.
+      - When C(state) is C(pure), only the members specified will exist,
```

Comment:
Should add another line saying `C(pure)` was added in Ansible 2.8.

Reply:
Thanks @jborean93, just added this a moment ago.